4_advanced_usage/40_node_properties.txt

```diff
@@ -0,0 +1,65 @@
+=== Node properties
+
+==== Defining properties
+
+Node properties can be found in the "properties" tab of each node in Rudder.
+
+Node properties can be modified using Rudder API, see http://www.rudder-project.org/rudder-api-doc/#api-Nodes-updateNodeProperties
+
+Currently a property is interpreted as a string value. So properties can be seen as a list of key=value pairs. Example: datacenter=Paris.
+
+==== Using properties
+
+You can use a node property almost everywhere in Rudder:
+- in directive parameters
+- in the technique editor
+- in your own techniques an generic methods code
+
+To use a property, simply use the variable node.properties with the variable call syntax.
+
+Example with a string property named datacenter:
+
+----
+
+${node.properties[datacenter]}
+
+----
+
```

Comment:
We should add a warning here:

Currently node properties can not be used inside ${eval } JavaScript blocks, since they are evaluated during poliocy generation and node properties are only made available in the agents. A workaround for this will be provided in a future version of Rudder.